internal/provisioner/plain/main.go

```diff
@@ -134,10 +134,18 @@ func main() {
 		setupLog.Error(err, "unable to parse bundle content server URL")
 		os.Exit(1)
 	}
-	bundleStorage := &storage.LocalDirectory{
+
+	localStorage := &storage.LocalDirectory{
 		RootDirectory: storageDirectory,
 		URL:           *storageURL,
 	}
+	httpLoader := storage.NewHTTP(
+		// TODO: Use a trusted CA to connect to bundle content URLs.
+		storage.WithInsecureSkipVerify(true),
+		storage.WithBearerToken(cfg.BearerToken),
```

Comment:
How would we start making progress on this todo and how blocking should we consider it for this PR?

Reply:
I think it's non-blocking for this PR, and I have a follow-up that solves it :)